lib/rmt/cli/main.rb

```diff
@@ -29,6 +29,9 @@ def sync
   desc 'systems', _('List systems')
   subcommand 'systems', RMT::CLI::Systems
 
+  desc 'scc', _('SCC sync commands')
+  subcommand 'scc', RMT::CLI::Scc
```

Comment:
would not `RMT::CLI:SCC` be better? for example we use `RMT::SCC` in this project

Reply:
That doesn't work nicely with Thor:

```ruby
class RMT::CLI::SCC < RMT::CLI::Base
```

Then:

```
> ./bin/rmt-cli scc
Commands:
  rmt-cli s c c sync-systems  # Forward registered systems data to SCC
```

`s c c` <--- this.

Actually, it would be possible to fix this by re-defining `self.subcommand_prefix` in the subcommand class:

```ruby
def self.subcommand_prefix
  'scc'
end
```

But instead let's rename the command to `rmt-cli systems scc-sync` -- I think that would be nicer. `rmt-cli systems` already exists and there only 1 subcommand there; there will be also just 1 subcommand in `rmt-cli scc`.

So `rmt-cli systems scc-sync` or `self.subcommand_prefix`?

Reply:
I like the idea with `rmt-cli systems scc-sync` more. because like that we also get rid of a bit misleading command name `sync` - sync was used to pull data from SCC and not to push it there

Reply:
> because like that we also get rid of a bit misleading command name `sync` - sync was used to pull data from SCC and not to push it there

Hmm, what? :thinking: 

No, `rmt-cli sync` remains untouched by all this. :slightly_smiling_face:

Reply:
yeah, that is what I mean that we use keyword `sync` for sync from SCC to RMT and not other way around. so `rmt-cli systems scc-sync` is better

Reply:
Ah, OK. :slightly_smiling_face:

Reply:
OK, the rename is done.